packages/help-center/src/components/help-center.tsx

```diff
@@ -48,10 +51,13 @@ const HelpCenter: React.FC< Container > = ( { handleClose, hidden } ) => {
 	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
 	const primarySiteId = useSelector( ( state ) => getPrimarySiteId( state ) );
 
-	useSelect( ( select ) => select( USER_STORE ).getCurrentUser() );
+	useSelect( ( select ) => ( select( USER_STORE ) as UserSelect ).getCurrentUser(), [] );
 
 	const currentSite = window?.helpCenterData?.currentSite;
-	const site = useSelect( ( select ) => select( SITE_STORE ).getSite( siteId || primarySiteId ) );
+	const site = useSelect(
+		( select ) => ( select( SITE_STORE ) as SiteSelect ).getSite( siteId || primarySiteId ),
+		[ siteId, primarySiteId ]
```

Comment:
A more optimized version could be to change this to `siteId || primarySiteId`? WDYT @noahtallen

Reply:
Make sense to me 👍